sklearn/metrics/_pairwise_distances_reduction/_argkminlabels.pyx

```diff
@@ -0,0 +1,404 @@
+
+from cython cimport floating, integral
+from cython.parallel cimport parallel, prange
+from libcpp.map cimport map as cmap, pair
+from libcpp.vector cimport vector
+from libc.stdlib cimport free
+
+cimport numpy as cnp
+
+cnp.import_array()
+
+from ...utils._typedefs cimport ITYPE_t, DTYPE_t
+from ...utils._typedefs import ITYPE, DTYPE
+from ...utils._sorting cimport simultaneous_sort
+import numpy as np
+
+from sklearn.utils.fixes import threadpool_limits
+
+cpdef enum WeightingStrategy:
```

Comment:
I think this file was accidentally committed.

Reply:
Good catch!